extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json

```diff
@@ -0,0 +1,78 @@
+              "description": "Journey AI engagement scores provide likelihoods of a customer engaging with a newly delivered message.",
+              "type": "object",
+              "properties": {
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailEngagementScores": {
+                  "title": "Email Engagement Scores",
+                  "description": "Engagement scores here are for email channel messages.",
+                  "type": "object",
+                  "properties": {
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/clickEngagementScore": {
+                      "title": "Click Engagement Score",
+                      "description": "This score represents the probability for a customer to click on links in an email.",
+                      "type": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/openEnagementScore": {
+                      "title": "Open Engagement Score",
+                      "description": "This score represents the probability of a customer to open an email.",
+                      "type": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/retentionLevel": {
+                      "title": "Retention Level",
+                      "description": "Retention level is a qualitative categorization of customer's likelihood of staying subscribed to the email list.",
+                      "type": "string"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/retentionPercentRank": {
+                      "title": "Retention Percent Rank",
+                      "description": "Retention percent rank is a percentile rank of the likelihood of staying subscribed to the email list.",
+                      "type": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/unsubscriptionProbability": {
+                      "title": "Unsubscription Probability",
+                      "description": "Unsubscription probability is the probability of a recipient to unsubscribe from email channel conditioned on email opened.",
+                      "type": "number"
+                    }
+                  }
+                },
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/engagementScoresInfo": {
+                  "title": "Engagement Scores Info",
+                  "description": "This information contains engagement scores model version and scoring date as a concatenated string, e.g. v1.3.0_2020-11-04.",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/journeyai-engagement-scores"
+    }
+  ]
```

Comment:
Please add "meta:status": "experimental". Once it is good to go to PROD, we can change the status to "stable".. Please add this to the other schema too.

Reply:
Done @mjasrotia. Thanks,